implementation/src/main/java/io/smallrye/faulttolerance/SecurityActions.java

```diff
@@ -17,43 +17,45 @@
 
 import java.lang.reflect.AccessibleObject;
 import java.lang.reflect.Field;
+import java.lang.reflect.GenericArrayType;
 import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.lang.reflect.ParameterizedType;
+import java.lang.reflect.Type;
+import java.lang.reflect.WildcardType;
 import java.security.AccessController;
 import java.security.PrivilegedAction;
 import java.security.PrivilegedActionException;
 import java.security.PrivilegedExceptionAction;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Predicate;
+
+import static java.util.Arrays.asList;
 
 /**
- *
  * @author Martin Kouba
  */
-final class SecurityActions {
+public final class SecurityActions {
```

Comment:
remove `public`

Reply:
Fixed in https://github.com/michalszynkiewicz/smallrye-fault-tolerance/pull/13